eng/common/TestResources/New-TestResources.ps1.md

```diff
@@ -412,6 +412,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OutFile
```

Comment:
This needs to be added to the bottom of the .ps1 file, if not already. This shouldn't be hand-authored. Let me know if you need help generating the doc file (there's a module you need, and need to hack the help a bit afterward).

Reply:
Or, if you'd rather, just wait till I make some updates anyway to defaulting this switch for our repo, in which case I'll be regenerating docs anyway.

Reply:
It is in the PS1 already - any pointers to generating the doc would be helpful, thanks!

Reply:
Then just leave this for now. I'll rerun it again later.